crates/oxc_linter/src/rules/oxc/no_accumulating_spread.rs

```diff
@@ -10,19 +10,33 @@ use oxc_macros::declare_oxc_lint;
 use oxc_semantic::SymbolId;
 use oxc_span::Span;
 
-use crate::{ast_util::is_method_call, context::LintContext, rule::Rule, AstNode};
+use crate::{
+    ast_util::{call_expr_method_callee_info, is_method_call},
+    context::LintContext,
+    rule::Rule,
+    AstNode,
+};
 
 #[derive(Debug, Error, Diagnostic)]
 enum NoAccumulatingSpreadDiagnostic {
     #[error("oxc(no-accumulating-spread): Do not spread accumulators in Array.prototype.reduce()")]
     #[diagnostic(severity(warning), help("It looks like you're spreading an `Array`. Consider using the `Array.push` or `Array.concat` methods to mutate the accumulator instead.\nUsing spreads within accumulators leads to `O(n^2)` time complexity."))]
-    LikelyArray(#[label] Span),
+    LikelyArray(
+        #[label("Accululator is spread here.")] Span,
```

Comment:
`Accumulating spread is here`? What's a `Accululator` 🤔

Reply:
oops good catch